path_supplier/graphhopper_path_supplier.go

```diff
@@ -0,0 +1,92 @@
+package path_supplier
+
+import (
+	"encoding/json"
+	"fmt"
+	m "github.com/harfmuf/geolocation-generator/model"
+	"io/ioutil"
+	"net/http"
+)
+
+// precision to around .0001 deg, which corresponds to max 10m
+const URL_FORMAT = "%s%s?point=%.4f,%.4f&point=%.4f,%.4f&vehicle=%s&key=%s&points_encoded=false"
```

Comment:
Is 4 decimal places API's limitation?

Reply:
Nope, just arbitrary precision. From what I can see in GraphHopper repo, this is type double so maybe we could increase to 1m. 
```suggestion
const URL_FORMAT = "%s%s?point=%.5f,%.5f&point=%.4f,%.4f&vehicle=%s&key=%s&points_encoded=false"
```

Reply:
API returns 6 decimals, thus I suggest to send the same amount